scripts/python/add_users_everyone_github_team.py

```diff
@@ -103,7 +80,34 @@ def run():
         print_stack_trace(message)
 
 
-print("Start")
-run()
-print("Finished")
-sys.exit(0)
+def main():
+    if len(sys.argv) == 2:
+        # Get the GH Action token
+        oauth_token = sys.argv[1]
+    else:
+        raise ValueError("Missing a script input parameter")
+
+    # Setup a transport and gql_client to interact with the GH GraphQL API
+    try:
+        transport = AIOHTTPTransport(
+            url="https://api.github.com/graphql",
+            headers={"Authorization": "Bearer {}".format(oauth_token)},
+        )
+    except Exception:
+        print_stack_trace("Exception: Problem with the API URL or GH Token")
```

Comment:
Spotted with @NickWalt01 that these aren't being propagated to the job failing. WIll fix separately though to keep this PR concise.

Reply:
propaged in what way? The intention is to print or print then bug out (cannot remember exactly at the moment)

Reply:
Propagated in the sense that the GitHub action will still pass.

[REF](https://github.com/moj-analytical-services/operations-engineering/actions/runs/4201831040/jobs/7289247720): this action passed, though in the logs an exception was raised, but only the print message was shown. Therefor the exceptions aren't propagated to the REPL, I believe

Reply:
Ah okay yeah that make sense, I cannot remember why I set it up that way, prob so we weren't constantly alerted by failures which are rate limited or timed out, etc. With the refactor prob want to remove that function or make it raise an error on exit